hokusai/development.yml

```diff
@@ -2,16 +2,18 @@ version: "2"
 services:
   metaphysics:
 {% include 'templates/docker-compose-service.yml.j2' %}
-    environment:
-      - MEMCACHED_URL=metaphysics-memcached:11211
+    command: ["yarn", "start"]
     env_file:
       - ../.env.shared
+      - ../.env.development
```

Comment:
* I'd really like to get this convention right, and currently it's a little confusing. E.g., our 2 next.js projects use `.env.development` files to store baseline development configs that target staging. I wish there was a nice, clear label for a "fully containerized artsy development stack." Then, we could call it `.env.artsylocal` (not actually a clear name).
* Ideally (for flexibility), `hokusai dev start` should continue to work within a single project, targeting staging for other dependencies. Is there a way to pass it a flag or file to override the usual settings for the "full" containerized development env?

Reply:
> currently it's a little confusing

Fair point and a good callout. I thought about adding it to `.env.hokusai` but that would mean we are introducing a new non-standard flavor of `.env.*` and I wasn't sure that's what we want.

This is a good opportunity to sort this out so I'll have to discuss this more and come back with something better.

---

My impression currently is that we want to have 3 forms of vars:
1. staging -- The default when running locally
2. localhost -- Optional when needing to point to localhost
3. hokusai -- Should be the default when running `hokusai dev ...`

Does ☝️ seem right or am I missing something/have the wrong impression?

Reply:
This is a bigger discussion so moved it to [here](https://artsy.slack.com/archives/CA8SANW3W/p1697210781648959)